jobs/config.json

```diff
@@ -10301,5 +10301,12 @@
     "sigOwners": [
       "sig-testing"
     ]
+  },
+  "pull-cluster-registry-bazel": {
```

Comment:
You'll need entries for both `pull-cluster-registry-bazel-build` and `pull-cluster-registry-bazel-test`

Reply:
Thanks! I noticed that in the test failure as well and was going to fix it when @BenTheElder suggested combining the tests. The names should match now.